### PR TITLE
Localdev base image command

### PR DESCRIPTION
**Pull request by Red-HAP**
Local koku-base image changed to have a benign command that exits successfully. The base image should not run in local dev.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ services:
       args:
         GIT_COMMIT: ${GIT_COMMIT-local-dev}
         PIPENV_DEV: "True"
+    command: "echo 'Base Image Should Not Run'"
```
Comment (maskarb) [inline docker-compose.yml:11]: Is this command really necessary? For me, this image never runs locally.

**Review by maskarb — COMMENTED** [review]
A question:

File: docker-compose.yml (hunk 1 above)
Comment (Red-HAP) [reply docker-compose.yml:11]: On fedora, it stays running as a second koku-server.

**Review by myersCody — APPROVED** [review]
LGTM!

**Pull request merged**